src/modules/commander/commander_params.c

```diff
@@ -713,3 +713,12 @@ PARAM_DEFINE_INT32(COM_FLIGHT_UUID, 0);
  * @group Mission
  */
 PARAM_DEFINE_INT32(COM_TAKEOFF_ACT, 0);
+
+/**
+ * Action after link has been regained
+ *
+ * @value 0 Go back to mode the UAV was in before the failsafe
```

Comment:
This could possibly be unsafe if the previous mode was a non-position controlled mode such as Manual or Acro. Would it not make sense to limit it to just modes that are position/velocity controlled?